Src/BootCamp.Chapter/Shop.cs

```diff
@@ -1,28 +1,22 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Collections.Generic;
+
+namespace BootCamp.Chapter
 {
     public class Shop
     {
-        private decimal _money;
-        public decimal GetMoney()
-        {
-            return _money;
-        }
-
-        private Inventory _inventory;
+        public Inventory Inventory { get; private set; }
+        public decimal Money { get; private set; }
 
         public Shop()
         {
-
+            Inventory = new Inventory();
         }
```

Comment:
I don't think this constructor is ever used, consider removing it.

Reply:
I've removed the unused ctor from Shop.